packages/kolibri-tools/lib/i18n/crowdin.py

```diff
@@ -392,6 +392,13 @@ def download_translations(branch, project, locale_data_folder):
         code = lang_object[KEY_CROWDIN_CODE]
         locale_dir_path = local_locale_path(lang_object, locale_data_folder)
         logging.info("\tExtracting {} to {}".format(code, locale_dir_path))
+
+        #clear out existing files in langugage directory
+        for file_name in os.listdr(locale_dir_path):
+            file_path = os.path.join(locale_dir_path, file_name)
+            if os.path.isfile(file_path):
```

Comment:
As we are concerned only with CSV files here, we probably want to check that it is a CSV file before we delete it, just in case there are other files in this folder we don't want to delete!

Another way to do this would be to use the glob module instead of os.listdir, then you could just list out any existing CSV files and remove those: https://docs.python.org/3/library/glob.html#glob.glob

Reply:
Oh alright, I'd look into that thank you.